### ISSUE TITLE
Chart display after deployment

### ISSUE THREAD

**Issue by UCKFA** [n3-charts/line-chart#441]
The chart looks bad after deploying my angular app.
I am attaching some images.
Good chart:
http://imgur.com/jBIu9je
Bad chart:
http://imgur.com/79m9C1A

Also the tooltip is shown under the chart for some reason( on the bad one ).

**Comment by lorem--ipsum** [comment]
Hi and thank for your interest ! Have you considered using v2 ?

**Comment by UCKFA** [comment]
I am using v2. I had no problems of this sort with v1.

**Comment by lorem--ipsum** [comment]
Okay then this is caused by the CSS not being imported properly.

**Comment by UCKFA** [comment]
Hmmm. I am going to try to fix this tomorrow and come back with an update. Please don't close the issue yet. I will make a complete post about everything I did to get here because there have been other issues also.

**Comment by lorem--ipsum** [comment]
Sounds good to me !

**Comment by UCKFA** [comment]
Back with the update. The CSS was not loaded, it's fixed now.

For some reason, after installing v2 with npm, I could not load the module. It said that it is not available, even though I have given the .js , .css paths in index.html.

To make it work, I moved the n3-charts folder from node_modules to bower_components. It worked, but the deployment was not working( bower didn't take the files into consideration ).

After this, I moved the LineChart.js and LineChart.css to my app/scripts and app/styles folders. Now it i working properly.

If anyone encounters these problems, just add the library manually to the project and everything should be fine.

@lorem--ipsum Thank you for your help!
